Cubical/Data/FinWeak/Base.agda

```diff
@@ -0,0 +1,59 @@
+{-# OPTIONS --safe #-}
+module Cubical.Data.FinWeak.Base where
+
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.Function
+
+open import Cubical.Data.Bool
+open import Cubical.Data.Nat using (ℕ; zero; suc)
+
+open import Cubical.Relation.Nullary
+
+private variable
+  n : ℕ
+
+-- The only difference between FinPure and FinData is that
```

Comment:
I don't believe that helps to unconfuse a reader. I guess what you are trying to say is:
"The difference between FinPure and Fin in Cubical.Data.FinData is that we only 
have ```zero : FinPure 1``` and not ```zero : Fin n```"
But that is a bit misleading, since one could believe after the comment, that this is the only interesting difference and FinPure is some alternative to Fin from FinData. Maybe you could just clarify a bit and add, that as a consequence, FinPure will be a proposition?
Also, please comment on applications of this and why (or in what situation) one would use the Fin here instead of the Fin from FinData.